operator/config/manager/manager.yaml

```diff
@@ -78,6 +69,7 @@ spec:
               cpu: 50m
               memory: 50Mi
               ephemeral-storage: 5Gi
```

Comment:
we do not require ephemeral-storage now as runner emptyDir volumemount is removed

Reply:
Done